apps/src/applab/designMode.js

```diff
@@ -190,7 +189,7 @@ function renderIconToString(value, element) {
   ctx.textAlign = 'center';
   ctx.fillStyle = element.getAttribute('data-icon-color');
   var regex = new RegExp('^' + ICON_PREFIX + 'fa-');
-  var unicode = '0x' + icons.unicode[value.replace(regex, '')];
+  var unicode = '0x' + window.dashboard.iconsUnicode[value.replace(regex, '')];
```

Comment:
Let's drop `window` here as well to track this dependency.  (This way if `/* global dashboard */` is removed, eslint will complain.)